stimuli_ci_generation/Dockerfile

```diff
@@ -13,5 +13,5 @@ EXPOSE 8080
 # Use gunicorn webserver with one worker process and 8 threads.
 # For environments with multiple CPU cores, increase the number of workers
 # to be equal to the cores available.
-CMD ["gunicorn", "--bind",  ":8080", "--workers",  "1",  "--threads", "8", "--timeout", "0", "server.main:app"]
+CMD ["gunicorn", "--bind",  ":8080", "--workers",  "1",  "--threads", "1", "--timeout", "0", "server.main:app"]
```

Comment:
NOTE to self & team: let's try to optimize this after performance testing code is ready